haystack/components/audio/whisper_remote.py

```diff
@@ -32,14 +40,19 @@ def __init__(
         **kwargs,
     ):
         """
-        Transcribes a list of audio files into a list of Documents.
-
-        :param api_key: OpenAI API key.
-        :param model: Name of the model to use. It now accepts only `whisper-1`.
-        :param organization: The Organization ID, defaults to `None`. See
+        Creates an instance of the RemoteWhisperTranscriber component.
+
+        :param api_key:
+            OpenAI API key.
+        :param model:
+            Name of the model to use. It now accepts only `whisper-1`.
+        :param organization: T
+            he Organization ID, defaults to `None`. See
```

Comment:
```suggestion
        :param organization: 
            The Organization ID. See
```

We don't want to document defaults that are evident from the method declaration (#7074).